composer/tools/composer_db_transfer.py

```diff
@@ -3524,7 +3532,7 @@ def _update_airflow_db_connection(self: typing.Any) -> None:
             "airflow connections add airflow_db "
             "--conn-type postgres "
             "--conn-login $SQL_USER "
-            "--conn-password $SQL_PASSWORD "
+            '--conn-password="$SQL_PASSWORD" '
```

Comment:
Should this be with an equal sign (`=`)? the other similar entries above don't come with it.

Reply:
This is correct and needed as SQL_PASSWORD might have '-' as its first character. If that happens, Airflow CLI considers the password as a separate option and option parsing fails.

Reply:
Got it. Thank you!